src/Training.AirBnb.Clone.Frontend/index.html

```diff
@@ -3,10 +3,12 @@
   <head>
     <meta charset="UTF-8">
     <link rel="icon" href="/favicon.ico">
+    <script type="module" src="/src/main.ts"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script type="module" src="/src/main.ts"></script>
```

Comment:
Why 2 sources files are added ? Also need to add script as last element of body 
![Uploading Screenshot 2023-12-26 at 16.36.04.png…]()